lib/ledger_sync/adaptors/quickbooks_online/adaptor.rb

```diff
@@ -13,7 +13,8 @@ class Adaptor < LedgerSync::Adaptors::Adaptor
         ROOT_SANDBOX_URI  = 'https://sandbox-quickbooks.api.intuit.com'
         API_RESOURCE_MAPPING_OVERRIDE = {
           LedgerSync::Expense => 'purchase',
-          LedgerSync::LedgerClass => 'class'
+          LedgerSync::LedgerClass => 'class',
+          LedgerSync::Preference => 'preferences'
```

Comment:
What do you think about naming it `Preferences` given that it is a collection of preferences?

Reply:
Yeah, that makes more sense.